instagram/src/components/CommentSection/commentsection.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Comment from '../CommentSection/Comment';
+import styles from './commentsection.css';
+import PropTypes from 'prop-types';
+
+const CommentSection = (props) =>{
+    return(
+        <div className = "comment-section">
+            {props.comments.map(comment =>{
+                return(
+                    <Comment key = {comment.key} username = {comment.username} text = {comment.text} />
```

Comment:
remember what we talked about in zoom: key isn't a prop that is handed down from Parent to child